### PR TITLE
Positioning of the startups in result page

### PR DESCRIPTION
**Pull request by Karyum**
rel #69

### CODE REVIEW

File: src/views/search_results.hbs [hunk 1]
```diff
@@ -61,7 +61,7 @@
 
     </form>
   </div>
-  <div class="fl w-69 mv5 ml3">
+  <div class="w-69 mv5 ml3">
```
Comment (esraajb) [inline src/views/search_results.hbs]: w-69?

Comment (Karyum) [inline src/views/search_results.hbs]: it needs to be just under the full width so everything would be showing on the page

Comment (esraajb) [inline src/views/search_results.hbs]: I'm not sure that w-69 exists in Tachyons

Comment (Karyum) [inline src/views/search_results.hbs]: ohhh yeah your right give me a sec it will be changed

Comment (Karyum) [inline src/views/search_results.hbs]: @esraajb changes have been made

**Pull request merged**